### PR TITLE
Jumbotron

### PR DESCRIPTION
**Pull request by MaeveScarry**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,24 +1,20 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Jumbotron fluid>
+        <Container>
+          <h1>Fluid jumbotron</h1>
```
[the remark this answers is not in the document]
Comment (MaeveScarry) [reply src/App.js]: False positive

File: src/App.js [hunk 2]
```diff
@@ -1,24 +1,20 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import Jumbotron from "react-bootstrap/Jumbotron";
+import Container from "react-bootstrap/Container";
+import "./App.css";
```
[the remark this answers is not in the document]
Comment (MaeveScarry) [reply src/App.js]: False positive

File: src/App.js [hunk 3]
```diff
@@ -1,24 +1,20 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import Jumbotron from "react-bootstrap/Jumbotron";
+import Container from "react-bootstrap/Container";
```
[the remark this answers is not in the document]
Comment (MaeveScarry) [reply src/App.js]: False positive

File: src/App.js [hunk 4]
```diff
@@ -1,24 +1,20 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import Jumbotron from "react-bootstrap/Jumbotron";
```
[the remark this answers is not in the document]
Comment (MaeveScarry) [reply src/App.js]: False positive

File: src/App.js [hunk 5]
```diff
@@ -1,24 +1,20 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
```
[the remark this answers is not in the document]
Comment (MaeveScarry) [reply src/App.js]: False positive

**Review by SonyaMoisset — APPROVED** [review]
LGTM :)

**Pull request merged**